setup.cfg

```diff
@@ -0,0 +1,20 @@
+[flake8]
```

Comment:
I don't want a setup.cfg, all shall be in pyproject.toml. Also I do not have that many extend-ignores in my setup. ;-)